src/Global/Checkbox.js

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment } from "react";
+
+function Checkbox(props) {
+  function handleEditTaskCompletionStatus() {
+    // use onChangeTaskStatus for changing task completion status on View Edit Create Task Page
+    props.onChangeTaskStatus && props.onChangeTaskStatus(!props.tempTaskStatus);
```

Comment:
Would be nice if you could have a single callback and ensure that whoever uses the checkbox will do the appropriate callback to either onChangeTaskStatus or onEditTask.